src/components/modal/modalHeader/modalHeader.tsx

```diff
@@ -7,15 +7,28 @@ const cx = classNames.bind(styles);
 
 interface ModalHeaderProps {
   title?: ReactNode;
+  headerContentClassName?: string;
   headerNode?: ReactNode;
+  headerDescription?: string;
```

Comment:
Looks like it can also be a react node
![image](https://github.com/user-attachments/assets/1dd66c7b-4e9b-47ad-8ce8-010aa7962b3d)
We can provide basic style for `description`, but in case it will be a react node, it will handle its internal styling without necessity of additional `headerContentClassName` props.